apis/atrium-api-cc-en_UK/src/main/kotlin/ch/tutteli/atrium/api/cc/en_UK/charSequenceAssertions.kt

```diff
@@ -217,3 +217,13 @@ fun <T : CharSequence> Assert<T>.isEmpty()
 @Deprecated("Use pendant from package en_GB; will be removed with 1.0.0", ReplaceWith("ch.tutteli.atrium.api.cc.en_GB.isNotEmpty()"))
 fun <T : CharSequence> Assert<T>.isNotEmpty()
     = addAssertion(AssertImpl.charSequence.isNotEmpty(this))
+
+/**
+ * Makes the assertion that [AssertionPlant.subject] [CharSequence].[kotlin.text.isNotBlank].
+ *
+ * @return This plant to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated("Use pendant from package en_GB; will be removed with 1.0.0", ReplaceWith("ch.tutteli.atrium.api.cc.en_GB.isNotBlank()"))
```

Comment:
td;lr please do not add new functionality to en_UK

I usually do not port functions back to en_UK and only add new functionality to en_GB. One who uses en_UK-bundle now and updates to the v0.8.0 en_UK-bundle in the future will get the function via the en_GB package as well => en_UK-bundle comprises both, deprecated functions from en_UK and en_GB (so that migration is easy).
Adding new functionality to en_UK as well increases maintenance.